recipes/oschmod/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "oschmod" %}
+{% set version = "0.3.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/oschmod-{{ version }}.tar.gz
+  sha256: bec99216f31615ee653b2a5c87cacfb4e4b6184c0e9f71da186300dacae974f3
+
+build:
+  entry_points:
+    - oschmod = oschmod.cli:main
+    - ochmod = oschmod.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pywin32-on-windows  # [win]
```

This package exist so packages that depend on `pywin32` on Windows can be noarch.

```suggestion
    - pywin32-on-windows
```